test/sharness/t0066-migration.sh

```diff
@@ -90,11 +90,11 @@ test_expect_success "ipfs repo migrate succeed" '
 
 test_expect_success "output looks good" '
   grep "Found outdated fs-repo, starting migration." migrate_out > /dev/null &&
-  grep "Success: fs-repo has been migrated to version 10" migrate_out > /dev/null
+  grep "Success: fs-repo has been migrated to version 11" migrate_out > /dev/null
```

Comment:
This looks like a duplicate of the test at line 72-74.

You may want to take this test from the current master branch and reapply your changes.

Reply:
This test is up to date with master branch, all the new code is only related this me PR.

This line looks similar to 71-74, cause it does test the same thing but it is the result of another command. In 71-74 we test the result of the `ipfs daemon` command, while here we test the result of the `ipfs migrate` command.